test/meson.build

```diff
@@ -1,11 +1,15 @@
 
+configure_file(input : 'pytest/data/wikibooks_be_all_nopic_2017-02.zim',
+               output : 'wikibooks_be_all_nopic_2017-02.zim',
+               copy: true )
```

Comment:
This is not a configuration file.
We should copy `test/pytest/data/wikibooks_be_all_nopic_2017-02.zim` to a directory like `test/data` and modify both test in pytest and gtest tests to use this file. 

If needed, we can use a define to set the path to the zim file in the source using meson.